src/Sarif/Core/Stack.cs

```diff
@@ -0,0 +1,166 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.
+// Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Reflection;
+using System.Text;
+using DotNetStackFrame = System.Diagnostics.StackFrame;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    public enum StackFormat
+    {
+        Default,
+        TrailingNewLine
+    }
+
+    /// <summary>
+    /// A call stack relevant to a SARIF result.
+    /// </summary>
+    public partial class Stack
+    {
+        private const string IN = " in ";
```

Comment:
I know the design guidelines only apply to public API surface, but... "screaming caps"? Would `In`, `At`, `Line` be ok?